android/libs/qtandroidserialport/qserialport_android.cpp

```diff
@@ -38,12 +38,12 @@ bool QSerialPortPrivate::open(QIODevice::OpenMode mode)
         return false;
     }
 
-    /*if (mode & QIODevice::ReadOnly) {
```

Comment:
What's with this uncommented and then commented out stuff?

Reply:
I tried to model this file after Qt's Unix class, but some commented out code doesn't work as I expected yet or I'm not sure it's necessary. I'll either fix it or remove it in a follow up after more testing.